pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -95,12 +98,20 @@ type ScrapeConfigSpec struct {
 	// HonorLabels chooses the metric's labels on collisions with target labels.
 	// +optional
 	HonorLabels *bool `json:"honorLabels,omitempty"`
+	// HTTP scheme to use for scraping.
+	// If empty, Prometheus uses the default value `http`.
+	// +kubebuilder:validation:Enum=http;https
```

Comment:
see below
```suggestion
	// +kubebuilder:validation:Enum=HTTP;HTTPS
```

Reply:
setting with capitals, prometheus throws error

```
ts=2023-06-15T06:03:34.480Z caller=scrape.go:499 level=error component="scrape manager" scrape_pool=scrapeconfig/default/scrape-config-scheme msg="Creating target failed" err="instance 0 in group 0: invalid scheme: \"HTTPS\""
```

Reply:
Should we convert this to lowercase in operator code? I found api validation not fully useful if we do have to convert again in code

Reply:
addressed here https://github.com/prometheus-operator/prometheus-operator/pull/5679#discussion_r1230552798